ctpolicy/chromepolicy.go

```diff
@@ -61,6 +62,44 @@ func (chromeP ChromeCTPolicy) LogsByGroup(cert *x509.Certificate, approved *logl
 	return groups, outerror
 }
 
+// LogsByGroup describes submission requirements for embedded SCTs according to
+// https://github.com/chromium/ct-policy/blob/master/ct_policy.md#qualifying-certificate.
+func (chromeP ChromeCTPolicy) LogsByGroup2(cert *x509.Certificate, approved *loglist2.LogList) (LogPolicyData, error) {
+	var outerror error
+	googGroup := LogGroupInfo{Name: "Google-operated", IsBase: false}
+	googGroup.populate2(approved, func(op *loglist2.Operator) bool { return op.GoogleOperated() })
+	if err := googGroup.setMinInclusions(1); err != nil {
```

Comment:
Seems like you save all the newly created `err` but don't take any other action, could you save some boilerplate and do something like this:
```golang
err = blahblah()
```
?

Reply:
Done